.vale/DevDocs/Contractions.yml

```diff
@@ -0,0 +1,30 @@
+# Recommends using contracted forms of verbs with 'not'
+extends: substitution
+message: "It's okay to use the contracted form '%s' instead of '%s'."
+level: suggestion
+ignorecase: true
+action:
+  name: replace
+swap:
+  are not: aren't
+  cannot: can't
```

Comment:
`can not`, no?

Reply:
No, cannot is a correct but not good spelling